descwl_shear_sims/shifts.py

```diff
@@ -129,3 +145,38 @@ def get_random_shifts(*, rng, dim, buff, size):
     shifts['dy'] = rng.uniform(low=low, high=high, size=size)
 
     return shifts
+
+
+def get_pair_shifts(*, rng, sep):
+    """
+    get a set of gridded shifts, with random shifts at the pixel scale
+
+    Parameters
+    ----------
+    rng: numpy.random.RandomState
+        The random number generator
+    sep: float
+        Separation of pair in arcsec
+
+    Returns
+    -------
+    shifts: array
+        Array with dx, dy offset fields for each point, in
+        arcsec
+    """
+
+    shifts = np.zeros(2, dtype=[('dx', 'f8'), ('dy', 'f8')])
+
+    angle = rng.uniform(low=0, high=2*np.pi)
```

Comment:
This angle should only be to pi and not 2pi.

Reply:
true, but it will give the same result

Reply:
My student and I found weird results using 2pi but maybe that was another bug.